File: webApi/Models/Vuelo.cs
using System;
using System.Collections.Generic;

namespace webApi.Models;

public partial class Vuelo
{
    public int Vueloid { get; set; }

    public string Codigoaeropuertoorigen { get; set; }

    public string Codigoaeropuertodestino { get; set; }

    public string Origen { get; set; }

    public string Destino { get; set; }

    public int? Viajeid { get; set; }

    public int? Avionid { get; set; }


}

File: AdminWebApp/src/app/Models/Vuelo.ts
export class Vuelo {
    vueloId: number;
    codigoAeropuertoOrigen: string;
    codigoAeropuertoDestino: string;
    origen: string;
    destino: string;
    viajeId: number;
    avionId: number;
  
    constructor(
      vueloId: number,
      codigoAeropuertoOrigen: string,
      codigoAeropuertoDestino: string,
      origen: string,
      destino: string,
      viajeId: number,
      avionId: number
    ) {
      this.vueloId = vueloId;
      this.codigoAeropuertoOrigen = codigoAeropuertoOrigen;
      this.codigoAeropuertoDestino = codigoAeropuertoDestino;
      this.origen = origen;
      this.destino = destino;
      this.viajeId = viajeId;
      this.avionId = avionId;
    }
  }